src/render.jl

```diff
@@ -136,13 +175,13 @@ function relpath(path::ByteString, startpath::ByteString = ".")
     prefix_num = findlast(x -> !isempty(x), start_arr) - i - 1
     if prefix_num >= 0
         prefix = pardir * Base.path_separator
-        relpath_ = isempty(pathpart)                                      ?
-            (prefix^prefix_num) * pardir                                  :
-            (prefix^prefix_num) * pardir * Base.path_separator * pathpart
+        relpath_ = isempty(pathpart)                                          ?
+                (prefix^prefix_num) * pardir                                  :
+                (prefix^prefix_num) * pardir * Base.path_separator * pathpart
```

Comment:
I changed it to this: otherwise we get over 100 char line

```
    sep = Base.path_separator
    pathpart = join(path_arr[i+1:findlast(x -> !isempty(x), path_arr)], sep)
    prefix_num = findlast(x -> !isempty(x), start_arr) - i - 1
    if prefix_num >= 0
        prefix = pardir * sep
        relpath = string(prefix^prefix_num, pardir, isempty(pathpart) ? "" : sep * pathpart)
    else
        relpath_ = pathpart
    end
```

but if you don't mind about 107 chars I can leave the: Base.path_separator